File: Beginner/1010 - Simple Calculate/1010.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);

        String[] input;
        input = in.readLine().trim().split(" ");
        int qtd1 = Integer.parseInt(input[1]);
        double valor1 = Double.parseDouble(input[2]);

        input = in.readLine().trim().split(" ");
        int qtd2 = Integer.parseInt(input[1]);
        double valor2 = Double.parseDouble(input[2]);

        double total = (qtd1 * valor1) + (qtd2 * valor2);
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", total);
    }
}

File: Beginner/1010 - Simple Calculate/1010.py
cod1, qtd1, valor1 = input().split(' ')
qtd1 = int(qtd1)
valor1 = float(valor1)

cod2, qtd2, valor2 = input().split(' ')
qtd2 = int(qtd2)
valor2 = float(valor2)

total = (qtd1 * valor1) + (qtd2 * valor2)
print(f"VALOR A PAGAR: R$ {total:.2f}")